lib/browser/api/auto-updater/auto-updater-win.js

```diff
@@ -1,71 +1,70 @@
 'use strict'
 
-const app = require('electron').app
-const EventEmitter = require('events').EventEmitter
+const { app } = require('electron')
+const { EventEmitter } = require('events')
 const squirrelUpdate = require('./squirrel-update-win')
-const util = require('util')
 
-function AutoUpdater () {
-  EventEmitter.call(this)
-}
-
-util.inherits(AutoUpdater, EventEmitter)
-
-AutoUpdater.prototype.quitAndInstall = function () {
-  if (!this.updateAvailable) {
-    return this.emitError('No update available, can\'t quit and install')
+class AutoUpdater extends EventEmitter {
+  constructor () {
```

Comment:
Yep, I lost internet before I could push from my laptop.  I was in the middle of a rebase 😢